stdlib/bytearray.mini

```diff
@@ -2,15 +2,10 @@
 // Copyright 2020, Offchain Labs, Inc. All rights reserved.
 //
 
-use std::expandingIntArray::ExpandingIntArray;
-use std::expandingIntArray::expandingIntArray_new;
-use std::expandingIntArray::expandingIntArray_get;
-use std::expandingIntArray::expandingIntArray_getConsecutive;
-use std::expandingIntArray::expandingIntArray_set;
-use std::expandingIntArray::expandingIntArray_setN;
+use std::random::RandomGenerator;
```

Comment:
This import is confusing since it doesn't appear to be used

Reply:
Added a comment explaining this.